Lib/test/support/__init__.py

```diff
@@ -1892,6 +1894,16 @@ def setswitchinterval(interval):
             interval = minimum_interval
     return sys.setswitchinterval(interval)
 
+def get_pagesize():
```

Comment:
I'll change https://github.com/python/cpython/blob/main/Lib/test/memory_watchdog.py#L13 to get_pagesize in other pr.